JS Basics/Strings/Problem_7/index.js

```diff
@@ -0,0 +1,12 @@
+function parseURL(str) {
+    const regex = /(.*):\/\/([^\/]+)(\/.*)$/i;
+    const matches = str.match(regex);
+    const protocol = matches[1];
+    const server = matches[2];
+    const resource = matches[3];
+    return console.log("protocol: " + protocol + "\nserver: " + server + "\nresource: " + resource);
+}
```

Comment:
Include QueryParameter parsing -> output them as a json object with the query parameter as a key and the value as a value.
![image](https://github.com/YulianaSTrifonova/internship/assets/92873843/1fa21bf1-f119-4bbf-9d2a-a51d771ea615)